tests/test.js

```diff
@@ -30,7 +30,8 @@ network:
 config_end: true
 `;
 
-const timeout = 15000;
+const timeout = 30000;
+const timeout_cleanup = 5000;
```

Comment:
Units are in ms?

Reply:
Yup
I'll add a comment - working on improving this tests as we just figured out how to turn off DHCP